.readthedocs.yaml

```diff
@@ -6,9 +6,8 @@ build:
     python: "3.12"
   jobs:
     post_install:
-      - pip install uv
-      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH uv pip install .[doc]
+      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH pip install .[doc]
```

Comment:
if you want pip, go back to this version instead: https://github.com/theislab/ehrdata/blob/14e3d923eda9e4fc1a7e2ee8c8c8158b3458ec65/.readthedocs.yaml

Reply:
just a check if not using uv makes the docs build work (it does for ehrapy) but I did not have much hope here this is the issue... was just for trying if a quick temp fix can get a docs build for a nice screenshot
keeping uv here & try to fix with it